docs/modules/ROOT/pages/release-notes/release-notes-24-0.adoc

```diff
@@ -0,0 +1,27 @@
+= Payara Enterprise 5.24.0 Release Notes
+
+== Supported APIs and Applications
+
+* Java EE 8 Applications
+* MicroProfile 3.3
+* Jakarta EE 8 Applications
```

Comment:
```suggestion
* Jakarta EE 8
* MicroProfile 3.3
```

If we passed the TCK, then we can write plainly that `Jakarta EE 8` is supported, not Jakarta EE 8 Applications. The phrase "Java EE 8 Applications" is there only for legal reasons because we can't say that `Java EE 8` itself is supported in Payara Server.

And I'd move Jakarta EE before MicroProfile because it's more relevant to Payara than MicroProfile.